.github/workflows/CI.yml

```
@@ -235,18 +238,18 @@ jobs:
         if: steps.cache-ledger-db.outputs.cache-hit != ''
         run: |
-          make BUILD_PROFILE=test import-headers
-          make BUILD_PROFILE=test import-nonces
+          make BUILD_PROFILE=test NETWORK=${{ matrix.network.name }} import-headers
+          make BUILD_PROFILE=test NETWORK=${{ matrix.network.name }} import-nonces
 
       - uses: actions/cache/save@v4
         if: github.event_name == 'push' || steps.cache-ledger-db.outputs.cache-hit == ''
         with:
-          path: ./ledger.db
+          path: ./ledger.${{ matrix.network }}.db
```

actionlint (1.7.7)

object, array, and null values should not be evaluated in template with ${{ }} but evaluating the value of type {magic: number; name: string}

(expression)